app/views/editLiability/disposeLiabilityBankDetails.scala.html

```diff
@@ -58,8 +58,7 @@
         "true" -> (messages("ated.label.yes"), Some("hidden-bank-details")),
         "false" -> (messages("ated.label.no"), Some("hidden-bank-details"))
         ),
-        '_legend -> messages("ated.bank-details.header"),
-        '_legendClass -> "visuallyhidden",
+        '_legend -> messages(""),
```

Comment:
This line can be completely removed rather than set empty.

Reply:
Strangely that what I thought as well, but if you remove the line, it doesn't show the radio buttons if you remove it completely.

Reply:
It worked when I did it. Did you accidentally delete part of the radio button stuff before it?